src/overlap_check/ProgressBar.hpp

```diff
@@ -2,23 +2,29 @@
 #ifndef DAGMC_PROGRESSBAR_H
 #define DAGMC_PROGRESSBAR_H
 
-#include <string>
-
 class ProgressBar {
 
  public:
   // constructor
-  ProgressBar();
+  ProgressBar() { 
+    // initialize bar
+    set_value(0.0);
+  };
 
   // destructor
-  ~ProgressBar();
+  ~ProgressBar()  {
+    if (need_final_newline) {
+      std::cout << "\n";
```

Comment:
maybe there we would want a `endl` to flush the buffer ?

Reply:
:+1: 
`\n` should be the same as `std::endl`, but `endl` may be safer across platforms.

Reply:
I thought `endl` would force the flush, where `\n` would not ?